src/api/app/views/webui2/webui/interconnects/new.html.haml

```diff
@@ -1,6 +1,54 @@
 :ruby
   @pagetitle = 'Interconnect'
+  opensuse_exists = Project.exists?(name: 'openSUSE.org')
+  tizen_exists = Project.exists?(name: 'tizen.org')
+  packman_exists = Project.exists?(name: 'Packman')
 
 .card.mb-3
   = render partial: 'tabs'
   .card-body
+    %h3 Connect a remote Open Build Service instance
+    %span Use a public OBS for default base distributions
+    .my-3
+      = button_tag('openSUSE', class: "interconnect mr-1 mb-1 btn #{opensuse_exists ? 'btn-outline-dark' : 'btn-outline-primary'}",
+                               data: { name: 'openSUSE.org',
+                                       remoteurl: 'https://api.opensuse.org/public',
+                                       title: 'Standard OBS instance at build.opensuse.org',
```

Comment:
I would also have the `title` on the button itself. I wonder if we should have a different title when the button is disabled. Something like `Active - Standard OBS instance at build.opensuse.org`. Right now, users have to guess that the disabled button means the interconnect is already created/active.

Reply:
Uhmmm, let's do it ^^

Reply:
Are you re-implementing check boxes? :wink:

Reply:
checkboxes :thinking:

Reply:
I'm not sure if this is the case, but once an interconnect is active, it cannot be deactivated/removed, right?

Reply:
it can, removing the project. As discussed in our meeting, let leave it as it is now and improve it later on.